scripts/githubActions/commitWhitespaceConventions.ps1

```diff
@@ -93,12 +93,4 @@ while($TRUE)
     }
 }
 
-if ($allMatched -eq $FALSE)
-{
-    "false" | Out-File -FilePath status_whitespace.txt -NoNewline
-    throw "Files contain tabs"
-}
-else
-{
-    "true" | Out-File -FilePath status_whitespace.txt -NoNewline
-}
+"true" | Out-File -FilePath status_whitespace.txt -NoNewline
```

Comment:
can you revert this?

Reply:
At the end for sure when I'll get this ready for review. I'll also need to revert the whitespace changes.  I just want faster feedback loop against gh actions without battling the checks.

Reply:
Reverted the change, seems that tabs in test cases is a required feature.